lib/charms/loki_k8s/v1/loki_push_api.py

```diff
@@ -2501,6 +2504,9 @@ def enable_endpoints(
         container: Container, active_endpoints: Dict[str, str], topology: JujuTopology
     ):
         """Enable forwarding for the specified Loki endpoints."""
+        if not container.can_connect():
+            return
+
```

Comment:
If we return here and skip this method, are we sure the method will run again after pebble is ready?

Reply:
Yes, because this method: `_PebbleLogClient.enable_endpoints()` is called from `enable_endpoints()` and this method is called from: `_update_logging()` and `_on_pebble_ready()`